### PR TITLE
@LACHAIN-362: Implemented the function

### PR DESCRIPTION
**Pull request by dushyantb**
https://latoken.atlassian.net/browse/LACHAIN-362

Function implemented and @soloway2010 has verified the same and working fine.

### CODE REVIEW

File: src/Lachain.Core/Blockchain/VM/ExternalHandler.cs [hunk 1]
```diff
@@ -308,6 +308,14 @@ public static void Handle_Env_WriteEvent(int signatureOffset, int valueOffset, i
             };
             frame.InvocationContext.Snapshot.Events.AddEvent(ev);
         }
+        
+        public static void Handler_Env_GetAddress(int resultOffset)
+        {
+            var frame = VirtualMachine.ExecutionFrames.Peek() as WasmExecutionFrame
+                        ?? throw new InvalidOperationException("Cannot call KECCAK256 outside wasm frame");
```
Comment (soloway2010) [inline src/Lachain.Core/Blockchain/VM/ExternalHandler.cs:315]: Perhaps "?? throw new InvalidOperationException("Cannot call GetAddress outside wasm frame");" is better?

File: src/Lachain.Core/Blockchain/VM/ExternalHandler.cs [hunk 2]
```diff
@@ -308,6 +308,14 @@ public static void Handle_Env_WriteEvent(int signatureOffset, int valueOffset, i
...
+            var result = (frame.CurrentAddress).ToBytes();
+            SafeCopyToMemory(frame.Memory, result, resultOffset);
```
Comment (soloway2010) [inline src/Lachain.Core/Blockchain/VM/ExternalHandler.cs:317]: Add returned value check?
```
if (!ret)
    throw new InvalidContractException("Bad call to (get_address)");
```

File: src/Lachain.Core/Blockchain/VM/ExternalHandler.cs (hunk 1 above)
Comment (dushyantb) [reply src/Lachain.Core/Blockchain/VM/ExternalHandler.cs:315]: yes... will update the code.

File: src/Lachain.Core/Blockchain/VM/ExternalHandler.cs (hunk 2 above)
Comment (dushyantb) [reply src/Lachain.Core/Blockchain/VM/ExternalHandler.cs:317]: No, as we are not returning any values here, and execution frame will always be having the `currentAddress`. So, essentially this check is not required .

**Review by soloway2010 — APPROVED** [review]

**Review by Skird — APPROVED** [review]

**Pull request merged**